src/main/java/io/fabric8/maven/docker/AbstractDockerMojo.java

```diff
@@ -54,7 +68,7 @@
     public static final String CONTEXT_KEY_PREVIOUSLY_PULLED = "CONTEXT_KEY_PREVIOUSLY_PULLED";
 
     // Minimal API version, independent of any feature used
-    public static final String API_VERSION = "1.18";
+    public static final String API_VERSION = "v1.18";
```

Comment:
Its correct without "v" because the "v" is added later to the version --> https://github.com/fabric8io/docker-maven-plugin/blob/e027a8abdeddf7e6a281a3f4b40ad6c3d95d6276/src/main/java/io/fabric8/maven/docker/AbstractDockerMojo.java#L292-L292